index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>James Kapella || Web Developer</title>
+    <link rel='stylesheet' href='normalize.css'>
+    <link rel='stylesheet' href='styles.css'>
+    <link href="https://fonts.googleapis.com/css?family=Oswald:200" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,300,400" rel="stylesheet">
+</head>
+<body>
+
+    <header>
+        <div class='container'>
```

Comment:
could the class of container be put on the header instead and the extra div removed?

Reply:
For this one, not easily, container div is a reactive element sitting inside the fixed header element, to make the container class work the same way on the header, i'd end up having to add several more lines of CSS to the header?

Reply:
(reactive in that it uses margin 0 auto to allow the margins to move a bit with different sized screens)